samples/batch/csharp/program.cs

```diff
@@ -6,136 +6,155 @@
+                    foreach (var transcription in paginatedTranscriptions.Values)
+                    {
+                        switch (transcription.Status)
+                        {
+                            case "Failed":
+                            case "Succeeded":
+                                // we check to see if it was one of the transcriptions we created from this client.
+                                if (!createdTranscriptions.Contains(transcription.Self))
+                                {
+                                    // not created form here, continue
+                                    continue;
+                                }
+                                completed++;
+
+                                // if the transcription was successful, check the results
+                                if (transcription.Status == "Succeeded")
+                                {
+                                    var paginatedfiles = await client.GetTranscriptionFilesAsync(transcription.Links.Files).ConfigureAwait(false);
+
+                                    var resultFile = paginatedfiles.Values.FirstOrDefault(f => f.Kind == ArtifactKind.Transcription);
+                                    var result = await client.GetTranscriptionResultAsync(new Uri(resultFile.Links.ContentUrl)).ConfigureAwait(false);
+                                    Console.WriteLine("Transcription succeeded. Results: ");
+                                    Console.WriteLine(JsonConvert.SerializeObject(result, SpeechJsonContractResolver.WriterSettings));
+                                }
+                                else
+                                {
+                                    Console.WriteLine("Transcription failed. Status: {0}", transcription.Properties.Error.Message);
+                                }
+                                break;
+
+                            case "Running":
+                                running++;
+                                break;
+
+                            case "NotStarted":
+                                notStarted++;
+                                break;
+                        }
+                    }
+
+                    // for each transcription in the list we check the status
+                    Console.WriteLine(string.Format("Transcriptions status: {0} completed, {1} running, {2} not started yet", completed, running, notStarted));
                 }
-                // </batchstatus>
+                while (paginatedTranscriptions.NextLink != null);
 
-                Console.WriteLine(string.Format("Transcriptions status: {0} completed, {1} running, {2} not started yet", completed, running, notStarted));
-                // check again after 5 seconds (can also be 1, 2, 5 min depending on usage).
-                await Task.Delay(TimeSpan.FromSeconds(5)).ConfigureAwait(false);
+                // </transcriptionstatus>
```

Comment:
>// </transcriptionstatus> [](start = 16, length = 25)

no <transcriptionstatus> defined before. Remove it?

Reply:
used in azure docs to inline docs